src/distributed-sql/dist-sql.md

```diff
@@ -0,0 +1,88 @@
+# Distributed SQL 
+
+Now we already know how the TiDB relational structure are encoded into the Key-Value form with versions.  In this section, we will focus on the following questions:
+
+* What happens when [TiDB] receives a SQL query?
+* How does [TiDB] excute SQL queries in a distributed way?
+  
+## What happens when [TiDB] receives a SQL query?
+
+Firstly, let's have a look at the following example:
+
+```
+   select count(*) from t where a + b  > 5;
+```
+
+![Figure 1](images/select_from_tidb.png)
+
+
+As the above figure discribes, when [TiDB] receives a SQL query from the client, it will process according the following steps:
```

Comment:
```suggestion
As the above figure describes, when [TiDB] receives a SQL query from the client, it will process according to the following steps:
```